src/main/java/com/openmrs/migrator/MigratorApplication.java

```diff
@@ -69,7 +72,7 @@ public void run(String... args) throws IOException {
     }
   }
 
-  private void runAllJobs() throws IOException {
+  private void runAllJobs() throws Exception {
```

Comment:
Why changing this to the base exception?

Reply:
created a settings exception that handles this

Reply:
SettingsException is already catching KettleException

Reply:
yes both are checked exceptions,  is  `SettingsException` handling  all **kettleProperties** configuration? if  it does it would be.  But Im not  sure if  KettleExeption also takes care of  jobs  and transformation  execution, if that does  we could   print  that in the _track trace_  may be  in two the separated  catch  blocks